src/utils.jsx

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+export function getArticles() {
+    return axios.get('https://nc-news-api-iv8w.onrender.com/api/articles')
```

Comment:
Here try to create an axios instance with a baseURL. For this see NC notes about axios.